### PR TITLE
feat: Add finished trading competitions

### PR DESCRIPTION
**Pull request by memoyil**

### CODE REVIEW

**Comment by memoyil** [comment]
@Chef-Yogi I updated with the new competition, Can you review this when you have time?

**Comment by Chef-Yogi** [comment]
sure, thanks😆

**Comment by memoyil** [comment]
@Chef-Cheems Can you provide me colorset for mobox competition banner in finished section

**Comment by Chef-Yogi** [comment]
@memoyil, Cheem is on holiday, maybe I can help.
I can get the data from Figma. 
and I am reviewing it now, this is a big PR😀

also, there will be a flexible SC for future TC, which can resolve the current situation we met (every TC need a new SC)

**Comment by Chef-Yogi** [comment]
Mobox banner bg 
radial-gradient(329.58% 50% at 50% 50%, #3B2864 0%, #191326 100%);

**Review by Chef-Yogi — DISMISSED** [review]

**Review by Chef-Yogi — DISMISSED** [review]

File: src/views/TradingCompetition/MoboxCompetition.tsx [hunk 1]
```diff
@@ -17,43 +16,40 @@ import {
...
 
 const MoboxCompetition = () => {
   const profileApiUrl = process.env.NEXT_PUBLIC_API_PROFILE
   const { account } = useWeb3React()
   const { t } = useTranslation()
+  const { isMobile } = useMatchBreakpoints()
   const { profile, isLoading } = useProfile()
   const { isDark, theme } = useTheme()
   const tradingCompetitionContract = useTradingCompetitionContractMobox(false)
   const [currentPhase, setCurrentPhase] = useState(CompetitionPhases.REGISTRATION)
```
Comment (chef-jojo) [inline src/views/TradingCompetition/MoboxCompetition.tsx:51]: over

Comment (memoyil) [reply src/views/TradingCompetition/MoboxCompetition.tsx]: Done

**Comment by Chef-Yogi** [comment]
@memoyil @chef-jojo  can we merge this to develop today?
I think It should merge into develop first and them I can do the work for MoD TC, otherwise, it will cause conflicts again.

**Comment by memoyil** [comment]
> @memoyil @chef-jojo can we merge this to develop today? I think It should merge into develop first and then I can do the work for MoD TC and flexible version TC, otherwise, it will cause conflicts again.

Imo It is ready to merge. Sonar is complaining about duplication but it will get fixed automatically when MoD work is done

**Review by chef-jojo — APPROVED** [review]

**Comment by chef-jojo** [comment]
Sonar check is not required.
Let's merge this. @Chef-Yogi you can follow up the changes

**Pull request merged**